app/transforms/dollar-cents.js

```diff
@@ -0,0 +1,13 @@
+import DS from 'ember-data';
+
+const { Transform } = DS;
+
+export default Transform.extend({
+  deserialize(serialized) {
+    return serialized  / 100;
+  },
+
+  serialize(deserialized) {
+    return Math.floor(deserialized * 100);
```

Comment:
`.floor` doesn't leave much room for it. Anything smaller than a cent is simply removed when posting to API. API deals with integers, so there will never be anything smaller than a cent sent to us.